src/PerformanceTests/Common/Scenarios/BaseRunner.cs

```diff
@@ -147,12 +239,14 @@ protected virtual Task Stop()
     }
 
 #if Version5
-    Task CreateOrPurgeQueues()
+    async Task CreateOrPurgeQueues()
     {
         var configuration = CreateConfiguration();
         if (IsPurgingSupported) configuration.PurgeOnStartup(true);
```

Comment:
I get why they are drained this way. What I don't get is why they are always drained? As far as I can remember, this was optional to drain queues?